lib/ruby_lsp/requests/formatting.rb

```diff
@@ -67,7 +67,7 @@ def run
         return if @formatter == "none"
 
         # Don't try to format files outside the current working directory
-        return unless @uri.sub("file://", "").start_with?(Dir.pwd)
+        return unless URI(@uri).path&.start_with?(T.must(WORKSPACE_URI.path))
```

Comment:
Yes, that might be a better way to go, but that is out of my perceived capability for now.
Aka: I don't think I could do this at this time. Especially because I am not familiar with the project structure and with sorbet.